common/extensions/api/_api_features.json

```diff
@@ -73,8 +73,7 @@
       "chrome://tab-strip/*",
       "chrome://wallet-panel.top-chrome/*",
       "chrome://wallet/*",
-      "chrome://brave-shields.top-chrome/*",
-      "chrome://vpn-panel.top-chrome/*"
+      "chrome://brave-shields.top-chrome/*"
```

Comment:
This is the one where I think we can just remove it altogether. No extension API access needed anymore 😄

Reply:
Fixed! perfect, thanks 😄